scriptmodules/supplementary/skyscraper.sh

```diff
@@ -241,8 +250,13 @@ function _scrape_skyscraper() {
     fi
 
 
+    # If 2nd parameter is set, use the cache for scrapin and not configured scrape source.
+    # Scraping from cache means we can omit '-s' from the parameter list.
+    if [[ -z $2 ]]; then
```

Comment:
please quote the `$2`

note the comment might read better if it was worded like "if 2nd parameter is not set blah, else use the cache for scraping", since the logic checks if it is set. minor but. oh and should it be scraping rather than scrapin ?

Reply:
OK, will do